collect_app/src/main/java/org/odk/collect/android/injection/config/AppDependencyComponent.java

```diff
@@ -322,4 +326,6 @@
     Scheduler scheduler();
 
     LocationClient locationClient();
+
+    Notifier notifier();
```

Comment:
I don't think this is being used anymore

Reply:
Right, fixed.